response_operations_ui/views/reporting_units.py

```diff
@@ -129,6 +129,12 @@ def resent_verification(ru_ref, email, party_id):
                             info='Verification email re-sent'))
 
 
+@reporting_unit_bp.route('response_chasing/<ce_id>', methods=['GET'])
```

Comment:
Should we have this link/the button in this PR at all?
Might be better saving it for the PR introducing reports so we don't have a broken button in master